model_analyzer/record/record.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, value, timestamp):
         timestamp : float
             Elapsed time from start of program
         """
+        assert type(value) is float
```

Comment:
Why is this a float? If the value is nanoseconds, it should not be a float.

Reply:
Previously it was in seconds and we used precision to represent everything after the decimal point to represent ms. I will fix this.